### DISCUSSION TITLE
Location of maps in macOS

### DISCUSSION THREAD

**Discussion by Raiden-GH** [organicmaps/organicmaps#6075] [Q&A] [answered]
Hello, 
I tested Organic Maps for iOS on macOS. 
The App works very well, but I can't find the location where the downloaded maps are stored. 
It would be great if someone here could help me.

**Comment by biodranik** [comment, marked as answer]
The path will look something like /Users/YOURUSERNAME/Library/Containers/SOME_RANDOM_UUID/Data/Documents/230901/Croatia_West.mwm

**Reply by Raiden-GH** [reply]
Thanks for the information! 
For me the _SOME_RANDOM_UUID_ folder is called _Organic Maps_.
